package.json

```diff
@@ -13,7 +13,9 @@
     "lint": "eslint . --cache --ext .js,.ts,.tsx,.jsx",
     "ts:lint": "tslint --project .",
     "####### Testing #######": "",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "airtap --local build/src/tests/*.test.js",
+    "browser-test": "airtap --loopback airtap.local --coverage *.test.js",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov",
```

Comment:
lets make sure these all work

Reply:
`"coverage"` seems to work

"browser-test" might not be producing coverage reports, though. the reports are empty. but I've also never run it when there are valid Tape tests